wallet/core/src/tx/generator/pending.rs

```diff
@@ -111,19 +127,19 @@ impl PendingTransaction {
     }
 
     /// Get UTXO entries [`Vec<UtxoEntryReference>`] of the pending transaction
-    pub fn utxo_entries(&self) -> &Vec<UtxoEntryReference> {
+    pub fn utxo_entries(&self) -> &AHashSet<UtxoEntryReference> {
```

Comment:
Can't as these are used with `contains()`.  In-fact, the reason `Vec<>` was converted to `HashSet<>` is to allow checking for a UTXO presence against the collection.  However, the UTXO ingest of primitives like the tx generator takes in utxo iterator, so that kind of addresses this but from another direction.

Reply:
Got you, change the comment above the function. It mentions `vector`

Reply:
good point.  done.